addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -460,14 +460,61 @@ body.editor_enable.editor_has_snippets {
             }
             .oe_snippet_thumbnail_img {
                 flex-shrink: 0;
-                width: 40px;
-                height: 32px;
+                width: 41px;
+                height: 30px; // 82x60 -> 41x30
                 padding: 0;
             }
-            .o_delete_btn {
+            .btn {
                 @extend %we-generic-link;
                 padding-top: 0;
                 padding-bottom: 0;
+                padding-left: 0;
+            }
+            &:not(:hover) .btn {
+                display: none;
+            }
+            we-input.o_we_user_value_widget {
+                cursor: pointer;
+
+                > div {
+                    display: flex;
+                    border: $o-we-sidebar-content-field-border-width solid $o-we-sidebar-content-field-border-color;
+                    border-radius: $o-we-sidebar-content-field-border-radius;
+                    background-color: $o-we-sidebar-content-field-input-bg;
+
+                    &:focus-within {
+                        border-color: $o-we-sidebar-content-field-input-border-color;
+                    }
+                    input {
+                        box-sizing: content-box;
+                        padding: 0 $o-we-sidebar-content-field-clickable-spacing;
+                        border: none;
+                        border-radius: 0;
+                        background-color: transparent;
+                        color: inherit;
+                        cursor: text;
+
+                        &:focus {
+                            outline: none;
+                        }
+                    }
+                    we-button {
+                        flex: 1 1 auto;
+                        line-height: 17px;
+                        text-align: center;
+
+                        &:hover {
+                            background-color: gray;
+                            cursor: pointer;
+                        }
+                        &.o_confirm_btn {
+                            color: $o-we-color-success,
+                        }
+                        &.o_cancel_btn {
+                            color: $o-we-color-danger,
+                        }
+                    }
+                }
```

Comment:
I did not pay attention to this in my first review :grimacing: Try not just copy/pasting scss rules like that, I know CSS is annoying but it's part of the code to maintain :wink: There is already scss code which styles we-input, at worst you need some *extra* rules for this specific one but not a copy of the other rules. Move the already existing scss rules at the right place for them to be applied on your input as well (or add the common rules in a placeholder rule and use the `@extend` directive like we do for other editor elements, probably the easiest to not break current rule priorities).

Also to color elements in green/red on hover, there already is a class for that, use it instead of duplicating the rule too, something like `o_we_text_...`

Reply:
Introduced an `@extend` but did not find an existing `:hover` style that matched my intent.